### PR TITLE
Fix a11y content issue on SuccessModal

### PR DESCRIPTION
**Pull request by nickoola**

### CODE REVIEW

File: src/successModal/SuccessModal.tsx [hunk 1]
```diff
@@ -58,15 +58,16 @@ class SuccessModal extends Component<SuccessModalProps> {
...
-        ariaLabelledBy={ariaLabelledBy}
-        ariaDescribedBy={ariaDescribedBy}
+        ariaLabelledBy={successContentId}
       >
         <img
           className={`${baseClassName}-bodyItem ${baseClassName}-image`}
           src={imageSrc}
           alt={imageText}
+          // This image is always decorative
+          aria-hidden={true}
```
Comment (mcellier) [inline src/successModal/SuccessModal.tsx]: Shouldn't we just put an empty `alt` then?

File: src/successModal/SuccessModal.tsx [hunk 2]
```diff
@@ -37,15 +38,14 @@ class SuccessModal extends Component<SuccessModalProps> {
       forwardedRef,
       imageSrc,
       imageText,
-      ariaLabelledBy,
-      ariaDescribedBy,
```
Comment (mcellier) [inline src/successModal/SuccessModal.tsx]: Breaking change?

File: src/successModal/SuccessModal.tsx (hunk 1 above)
Comment (nickoola) [reply src/successModal/SuccessModal.tsx]: That's what we do with the default value. I let the possibility to add a description (SEO purpose, image not loaded...).

File: src/successModal/SuccessModal.tsx (hunk 2 above)
Comment (nickoola) [reply src/successModal/SuccessModal.tsx]: No previously those props were not well used and not defined in the render at the end.

**Pull request merged**